tx_test.go

```diff
@@ -433,6 +433,169 @@ func TestTx_DeleteBucket_NotFound(t *testing.T) {
 	}
 }
 
+func TestTx_MoveSubBucket(t *testing.T) {
+	testCases := []struct {
+		name                     string
+		subBucketName            string
+		subBucketKey             string
+		subBucketValue           string
+		dstBucketName            string
+		subBucketExistRootBucket bool
+		subBucketExistDstBucket  bool
+		keyNoSubBucketDstBucket  bool
+		expErr                   error
+	}{
+		{
+			"happy path",
+			"subBucket",
+			"this is subBucket key",
+			"this is subBucket value",
+			"dstBucket",
+			true,
+			false,
+			false,
+			nil,
+		},
+		{
+			"subBucket not exist in root bucket",
```

Comment:
@fuweid  @ahrtr  I added these tests to `tx_text`, i adapted the code to test only the case when the `srcBucket` is nil, since these cases are covered by `bucket_test` 

ptal